streams/lb.m3u

```diff
@@ -22,6 +22,8 @@ http://185.105.4.236:1935/live/ngrp:livestream_all/live.m3u8
 https://live.3cd.io/v1/broadcast/mtv/playlist.m3u8
 #EXTINF:-1 tvg-id="NabaaTV.lb",Nabaa TV (720p) [Not 24/7]
 http://31.14.40.237:1935/live/Nabaa/playlist.m3u8
```

Comment:
31.14.40.237 -> 5dc7d824154d0.streamlock.net

Reply:
Isn't it better to remove the direct ip url?